src/components/Datathon/Leaderboard/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { connectDB } from '@/db/config';
+import React from 'react';
+import Leaderboard, { ITeamData } from '@/db/models/leaderboard';
+
+export default async function LeaderboardTable() {
+  connectDB();
+
+  const leaderboards = await Leaderboard.find({})
+    .sort({ timestamp: -1 })
+    .limit(1)
+    .exec();
+  const latestLeaderboard = leaderboards[0];
+  latestLeaderboard.data.sort((a: ITeamData, b: ITeamData) => {
+    if (a.score > b.score) {
+      return -1;
+    } else if (a.score < b.score) {
+      return 1;
+    } else {
+      return 0;
+    }
+  });
```

Comment:
None of the above code should exist here, should be moved to an `api/leaderboard` route instead